compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -851,7 +872,16 @@ public BType readType() throws IOException {
                                 names.fromString(funcName), env.pkgSymbol.pkgID, funcType,
                                 env.pkgSymbol, Symbols.isFlagOn(objFlags, Flags.NATIVE));
                         invokableSymbol.retType = funcType.retType;
-                        objectSymbol.methodScope.define(invokableSymbol.name, invokableSymbol);
+//                        objectSymbol.methodScope.define(invokableSymbol.name, invokableSymbol);
```

Comment:
Are we going to cleaning these stuffs up?

Reply:
yes, will clean all these once BIR symbol enter is fully functional, and it is almost there, only constant, services and resources are there to be finalized

Reply:
ack